make-util.js

```diff
@@ -1792,10 +1792,8 @@ var getBuildConfigGenerator = function (baseConfigToolPath) {
         configToolBuildUtility = path.join(baseConfigToolPath, "dev.sh");
     }
 
-    if (!fs.existsSync(programPath)) {
-        console.log(`BuildConfigGen not found at ${programPath}. Starting build.`);
-        run(configToolBuildUtility, true);
-    }
+    // build configToolBuildUtility if needed.  (up-to-date check will skip build if not needed)
+    run(configToolBuildUtility, true);
```

Comment:
If the build will be on every run now, could you please also remove the [--rebuild argument](https://github.com/microsoft/azure-pipelines-tasks/blob/master/make.js#L923)?

Reply:
Removed